FAQs/README.md

```diff
@@ -12,6 +12,8 @@
  4. [While installing AzTS solution I have provided my preferences for telemetry collection i.e. anonymous AzTS usage data and organization/team contact details. How do I update my preferences now?](#4-while-installing-azts-solution-i-have-provided-my-preferences-for-telemetry-collection-ie-anonymous-azts-usage-data-and-organizationteam-contact-details-how-do-i-update-my-preferences-now)
 5. [On running the Autoupdater function I am encountering an error related to conflicts in function runtime version.](#5-on-running-the-autoupdater-function-i-am-encountering-an-error-related-to-conflicts-in-function-runtime-version)
 6. [What are the recommended authentication settings for all App services or Function Apps?](#6-what-are-the-recommended-authentication-settings-for-all-app-services-or-function-apps)
+7. [How can I migrate classic Application Insights to Workspace-based? If I migrate Application Insights AzSK-AzTS-AppInsights-xxxxx to Workspace-based, will there be any impact to AzTS scan?](#7-how-can-i-migrate-classic-application-insights-to-workspace-based-if-i-migrate-application-insights-azsk-azts-appinsights-xxxxx-to-workspace-based-will-there-be-any-impact-to-azts-scan)
```

Comment:
same as above, it seems we ended the statement in mid way. Workspace-based resource seems better.